azerothcore/azerothcore-wotlk
# fix(Script/Spell): Gryphon/Wyvern Pet - Mounting Check Aura

* cherry-pick commit (https://github.com/TrinityCore/TrinityCore/commit/5f71c40b969c07f950e3dceb99037c7685512e71)

## Changes Proposed:
-  Implement spell Gryphon/Wyvern Pet - Mounting Check Aura

## Tests Performed:

- Tested in-game, works

## How to Test the Changes:


2. .add 49662
3. Learn the pet
4. Summon the pet
5. See that if you are not mounted, it will run with you
6. see that when you are mounted, it will start to fly


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
